.github/workflows/buildAndTest.yml

```diff
@@ -58,59 +58,10 @@ jobs:
         cache-suffix: ${{ matrix.os-arch }}-${{ matrix.llvm-build }}-${{ matrix.torch-binary }}
 
     - name: Configure os-arch='ubuntu-x86_64' llvm-build='in-tree' torch-binary='${{ matrix.torch-binary }}'
```

Comment:
Also, `llvm-build='in-tree'` should probably change to `llvm-build='${{ matrix.llvm-build }}'`